plugins/ufm_events_grafana_dashboard_plugin/build/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:22.04
+
+LABEL maintainer="anana@nvidia.com"
+
+ARG PLUGIN_NAME
+ARG BASE_PATH=/opt/ufm/ufm_plugin_${PLUGIN_NAME}
+ARG SRC_BASE_DIR=${PLUGIN_NAME}_plugin
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY ${SRC_BASE_DIR}/ ${BASE_PATH}/${SRC_BASE_DIR}/
+COPY ${SRC_BASE_DIR}/scripts/ /
+
+
+# Run installation scripts
+RUN /install_fluentd.sh
+RUN /install_loki.sh
+RUN /install_grafana.sh
+
+# Install Supervisor
+RUN apt-get update && apt-get install -y supervisor vim tzdata
+
+# move /etc/supervisor from the /etc, /etc dir will be overridden by the shared volume
+RUN mv -r /etc/supervisor /
```

Comment:
pelase dont set it in / root you can move it to /var/etc

Reply:
What is the difference?